src/store/sagas/swap.ts

```diff
@@ -259,7 +259,7 @@ export function* handleSwap(): Generator {
       pair: new Pair(tokenFrom, tokenTo, PAIRS[networkType][poolIndex].feeTier),
       xToY: isXtoY,
       amount: byAmountIn ? amountIn : amountOut,
-      knownPrice: knownPrice,
+      estimatedPriceAfterSwap: knownPrice,
```

Comment:
please change `knownPrice` name. It doesn't make sense now.

Reply:
done